src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -102,18 +100,21 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
 
         if (triggerAction == TDTM_Runnable.Action.AfterInsert &&
             !recursionAfterInsertCheck) {
-                handlesAfterInsertUpdate(newContacts, oldContacts, triggerAction, accountId);
+                handlesAfterInsertUpdate(newContacts, oldContacts, triggerAction, NULL);
```

Comment:
@lifewithryan , the reason why I'm passing NULL on handlesAfterInsertUpdate() inside line 101, is because this is specifically for AfterInsert process. AfterInsert and AfterUpdate process are calling the same method handlesAfterInsertUpdate() which then handles 1 for loop for contact because both AfterInsert and AfterUpdate share some processes and adds to the same list. Therefore, I didn't want to have to separate that to create the same redundant logic check. Once it is inside handlesAfterInsertUpdate() method, it loops through the contact, does the conditional logic check that is shared between AfterUpdate and AfterInsert, then it does another check for which state it is, either AfterUpdate or AfterInsert, then goes and calls HandleInsertWrapperLogic or HandleUpdateWrapperLogic. For this bug of contacts being marked deceased, I'm assuming that it should only occur after it's updated? I guess, a user could also insert contacts that are already deceased. I can update line 103, to not pass NULL to cover that scenario. Let me know if you're following and if you agree that I should cover that scenario as well.